src/components/Publish/_utils.ts

```diff
@@ -517,11 +511,16 @@ export async function getFeesPublishDDO(
   )
 
   const gasPrice = await web3.eth.getGasPrice()
+  const userBalance = await web3.eth.getBalance(accountId)
 
   const gasFeeEth = web3.utils.fromWei(
     (+gasPrice * +gasEstimate).toString(),
     'ether'
   )
+  const balanceEth = web3.utils.fromWei(userBalance.toString(), 'ether')
```

Comment:
same here with different native tokens on different chains, most likely we have to adapt this here too